vue.config.js

```diff
@@ -1,5 +1,24 @@
+const CompressionPlugin = require("compression-webpack-plugin");
+
 module.exports = {
   devServer: {
-    proxy: process.env.BASE_URL
+    proxy: process.env.BASE_URL,
+    port: 8000
+  },
+  productionSourceMap: false,
+  configureWebpack: config => {
+    if (process.env.NODE_ENV === "production") {
+      config.plugins.push(
+        new CompressionPlugin({
+          deleteOriginalAssets: true
+        })
+      );
+    }
+  },
+  chainWebpack: config => {
+    if (process.env.NODE_ENV === "production") {
+      config.plugins.delete("prefetch");
```

Comment:
If we add something like `"prefetch-src 'self'; "` should be able to turn this back on. More of a note.

Reply:
Its true and could be a small performance boost.